c14h/sql.go

```diff
@@ -27,8 +27,10 @@ func OpenDB() (err error) {
 
 func (v *Vortrag) Put() (err error) {
 	log.Println("Put", v)
-	tx, _ := db.Begin()
-
+	tx, err := db.Begin()
+	if err != nil {
+		reutrn err
```

Comment:
There is a typo here, most likely the cause for the CI-failure, other than that, it looks ready to merge for me.